department-of-veterans-affairs/caseflow
# Queue | Add Issue

Resolves #4477

### Description
Allows the user to add an issue to the case they're checking out

### Acceptance Criteria
- [x] Verify that when a user clicks the "Add issue" on the Select Dispositions screen they are taken to a page called "Add issue"
- [x]  Verify that under "Add issue" there are the following dropdown labeled "Select Program" and "Select Issue'
-  [x] Verify that there is a sub section called "Subsidiary Questions or Other Tracking Identifier(s)"
    **Note: These dropdown are conditional. For example, some issues only let you select two levels others three. This logic can be found here:
[Appendix F - VACOLS Issue Logic](https://github.com/department-of-veterans-affairs/appeals-design-research/blob/master/Projects/BVA%20Work%20Queue/Research/Documents/Appendix%20F%20-%20VACOLS%20Issue%20Coding.pdf)
  - Verify that there are conditional dropdown for Levels 1 to 3 
- [x]  Verify that after Level 3 there is a notes Input field
- [x] Verify that there is a link to the bottom left of the canvas with the following copy "Go back to Select Disposition" and takes them to the Select Dispositions screen 
- [x] Verify that there is a button on the bottom right called "Add issue" 
- [x] Verify that it adds the new issue below the last issue in the Select Dispositions screen #4471 
- [x] Verify that new issues are added to VACOLS

### Testing Plan
1. Go to Draft Decision checkout flow
2. Add Issue
3. Confirm issue appears after clicking Save
4. Without finishing checkout flow, reload, verify new issue appears
